src/test/java/redis/clients/jedis/commands/unified/pooled/PooledCommandsTestHelper.java

```diff
@@ -4,7 +4,7 @@
 
 public class PooledCommandsTestHelper {
 
-  private static final EndpointConfig nodeInfo = HostAndPorts.getRedisEndpoint("standalone0");
+  public static final EndpointConfig nodeInfo = HostAndPorts.getRedisEndpoint("standalone0");
```

Comment:
```suggestion
  static final EndpointConfig nodeInfo = HostAndPorts.getRedisEndpoint("standalone0");
```

_package private_ could be enough. But please make sure :)

Reply:
used also in different package - redis.clients.jedis.commands.unified.pipeline